corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py

```diff
@@ -1312,17 +1322,19 @@ def testCacheInvalidation(self):
         self.assertTrue(self.sync_log.phone_is_holding_case(case_id))
 
     def testCacheNonInvalidation(self):
+        device_id = 'AXDPWO2NZYCBPLSTPBAWGVH9'
         original_payload = RestoreConfig(
             project=self.project,
             restore_user=self.user,
             params=RestoreParams(
                 version=V2,
                 sync_log_id=self.sync_log._id,
+                device_id=device_id,
             ),
             **self.restore_options
         ).get_payload().as_string()
         self.sync_log = get_properly_wrapped_sync_log(self.sync_log._id)
-        self.assertTrue(has_cached_payload(self.sync_log, V2))
+        self.assertTrue(has_cached_payload(self.sync_log, device_id))
```

Comment:
How did we not get a merge conflict here when #17468 was merged? Looks like the changes to this test have been overwritten by that PR.